index.html

```diff
@@ -24,15 +24,45 @@
   <body>
     <header class="header">
       <nav id="navbar" class="navbar">
-        <a href="">Kennedy Owusu</a>
-        <a id="menu-icon" href=""><img src="./images/nav-icons/nav-icon.png"></a>
+        <a class="navbar-title" href="">Kennedy Owusu</a>
         <ul>
           <li><a class="header-nav" href="">Portfolio</a></li>
           <li><a class="header-nav" href="">About</a></li>
           <li><a class="header-nav" href="">Contact</a></li>
           <li><a class="header-nav" href=""><img src="./images/nav-icons/mail-icon.png"></a></li>
         </ul>
+
+      </nav>
+
+      <nav id="mobile-navbar">
+        
+        <div>
+          <a href="#" class="navbar-title" id="navbar-title">Kennedy Owusu</a>
+
+          <div class="hamburger">
```

Comment:
- [ ] The issue that the previous reviewer mentioned is still present, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/202138113-17f0242a-42e2-401c-9c38-916a02ea8707.png)